hooks/tk-nuke_actions.py

```diff
@@ -157,7 +157,7 @@ def _import_script(self, path, sg_publish_data):
         :param sg_publish_data: Shotgun data dictionary with all the standard publish fields.
         """
         import nuke
-        if not os.path.exists(path):
+        if not os.path.exists(path.decode('utf-8')):
```

Comment:
please add a comment pointing out that this is required for unicode paths on windows, however the nuke API itself handles utf-8 encoded paths correctly.